packages/amplify_authenticator/lib/src/mixins/authenticator_radio_field.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:amplify_authenticator/src/widgets/authenticator_input_config.dart';
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+import 'package:flutter/foundation.dart';
+
+import 'package:flutter/material.dart';
+
+mixin AuthenticatorRadioField<FieldType, FieldValue,
+        T extends AuthenticatorFormField<FieldType, FieldValue, T>>
+    on AuthenticatorFormFieldState<FieldType, FieldValue, T>
+    implements SelectableConfig<FieldValue> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget buildFormField(BuildContext context) {
+    final inputResolver = stringResolver.inputs;
+    onChanged?.call(selectionValue!);
+    return Column(
+      children: <Widget>[
+        if (selections.isNotEmpty)
```

Comment:
*nit:* Is this needed? Wouldn't this just be an empty list if selections was empty?

Reply:
I'm hoping to avoid this once I iterate on the radio buttons, but currently it's in place due the fact that we generate these at runtime.